api/v1beta1/cinderapi_types.go

```diff
@@ -17,35 +17,73 @@ limitations under the License.
 package v1beta1
 
 import (
+	condition "github.com/openstack-k8s-operators/lib-common/pkg/condition"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // CinderAPISpec defines the desired state of CinderAPI
 type CinderAPISpec struct {
-	// CR name of managing controller object to identify the config maps
-	ManagingCrName string `json:"managingCrName,omitempty"`
-	// Cinder Database Hostname String
-	DatabaseHostname string `json:"databaseHostname,omitempty"`
-	// Cinder Scheduler Container Image URL
+	// TODO: This legacy field here and in other CRs could probably be handled
+	// through labels placed on the CR by the parent Cinder CR/controller itself
+	// +kubebuilder:validation:Required
+	// ManagingCrName - CR name of managing controller object to identify the config maps
+	ManagingCrName string `json:"managingCrName"`
```

Comment:
I know we used this in cnosp, but I don't think we need this any longer. we can get the owning CR from the owner reference.

Reply:
Right, I had placed a comment there to a similar effect.  I can clean this up

Reply:
Done